prime-router/docs/design/design/data-model.md

```diff
@@ -0,0 +1,355 @@
+
+#### `item_lineage` table
+
+The `item_lineage` table helps keep track of individual Report [Items](../../universal-pipeline/README.md#report-and-item) as they flow through the Universal Pipeline and get split into different, and potentially multiple, Reports. Similar to the `report_lineage` table, the `item_lineage` table contains the id of the parent and child Report, and in addition, the parent and child index. The index indicates what Item in a Report the `item_lineage` refers to. For example, if `parent_index` is 2 and `child_index` is 3, that means the second item in the parent Report file is the third item in the child Report file. There is no `item` table, a decision that was made during the initial design, so items only show up in the database via the `item_lineage` table for subsequent child Reports. Also like `report_lineage`, `item_lineage` is stored as a graph and can be queried with Postgres' recursive [Common Table Expressions](https://www.postgresql.org/docs/current/queries-with.html) feature.
+
+#### Examples
+
+A good set of examples on how to query report and item lineage was created for the CDC ELIMS Pilot effort. These queries can be found in the repository [here](../../../examples/cdc-elims-metabase-queries/README.md).
+
+### Lookup Tables
+
+The Lookup Tables feature allows ReportStream to store and version datasets. Lookup Tables are generic and can be used for anything, but are mainly used for storing [LIVD tables](https://www.cdc.gov/csels/dls/livd-codes.html), so they can be referenced when validating or translating Reports.
+
+> Instructions for the most common use case of Lookup Tables, updating LIVD tables, can be found [here](../../getting-started/standard-operating-procedures/update-livd-table.md).
+
+#### Associated Tables
+
+- lookup_table_version
+- lookup_table_row
+
+#### `lookup_table_version` table
+
+`lookup_table_version` is the database table where the various Lookup Tables are _defined_. Each row in this table represents a particular version of a particular Lookup Table. For example, the report below shows 5 tables where the table named `fhirpath_filter_shorthand` has two versions. The old version, 1, is set to inactive and the newest version, 2, is active.
+
+![lookup-table-version](../../assets/lookup-table-version.png)
+
+#### `lookup_table_row` table
+
+The `lookup_table_row` table is responsible for storing the individual rows of a Lookup Table.
+
+![lookup-table-row](../../assets/lookup-table-row.png)
+
+`lookup_table_row_id` is the internal unique identifier of the row, whereas `lookup_table_version_id` links to the `lookup_table_version_id` in the `lookup-table-version` table -- that is to say this column identifies which specific Lookup Table and version of that Lookup Table the row belongs to. 
+
+`row_num` indicates the "virtual" index of the row in the Lookup Table, meaning this logical implementation of a table via two database tables isn't actually a single table in memory.
```

Comment:
I have read this a few times and am still confused. Can you please add further clarification? How is it a virtual index? How does that mean that it is two tables. Which two tables are you referring to?

Reply:
Updated! Let me know if still not clear and I can take another pass at it!

Reply:
This is outdated, please see updated version.

Reply:
I still am not seeing where it names the two tables here.